src/features/analysis/time-series/time-series-tooltip.tsx

```diff
@@ -70,6 +73,19 @@ function TimeSeriesTooltip({
         </span>
       </div>
 
+      {/* SMA row - only when SMA is enabled and value exists */}
+      {smaValue !== undefined && smaValue !== null && formatter && (
+        <div className="flex items-baseline justify-between gap-4 mt-2 pt-2 border-t border-slate-700/30">
+          <span className="text-xs flex items-center gap-1.5">
+            <span className="inline-block w-3 h-0.5 bg-orange-500/70 rounded-full" style={{ borderStyle: 'dashed' }} />
+            <span className="text-slate-400">SMA Trend</span>
```

Comment:
This is cool to see, though lets make this label state "Moving Average" or something similar (ie whatever you come up with to address the intentful naming comment). We've got a lil more room in the tooltip to provide clearer terms for users who won't be familiar with app (or industry) jargon.

<img width="309" height="223" alt="Image" src="https://github.com/user-attachments/assets/b28745dc-80de-4b6e-82d3-9db4b725bafb" />